kstreams/poc-ddd-aggregates/src/main/java/com/github/hpgrahsl/kafka/StreamingAggregatesDDD.java

```diff
@@ -0,0 +1,100 @@
+package com.github.hpgrahsl.kafka;
```

Comment:
Could we rename it into `io.debezium.examples.aggregation`?

Reply:
dito